src/pip/_internal/operations/install/wheel.py

```diff
@@ -52,6 +52,10 @@
 
 def normpath(src, p):
     # type: (str, str) -> str
+    # for Windows: if paths start with different disk symbols,
+    # do not handle reletive paths
+    if src[0] != p[0]:
+        return src.replace(os.path.sep, '/')
```

Comment:
Is it possible for either `src` or `p` be relative? This might result in weird edge cases in those. Would `os.path.splitdrive()` help here?

Reply:
I think all the paths would be absolute. But yes, `splitdrive()` can be a better approach.